.travis.yml

```diff
@@ -19,3 +54,6 @@ branches:
   only:
     - master
     - feature/laravel-5
```

Comment:
Could you please remove this one? It's an odd branch, I don't event remember how it got there.